internal/dag/builder_test.go

```diff
@@ -71,14 +86,17 @@ func TestDAGInsertGatewayAPI(t *testing.T) {
 
 	gatewayWithSelector := &gatewayapi_v1alpha1.Gateway{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "gatewayWithSelector",
-			Namespace: "default",
+			Name:      "contour",
+			Namespace: "projectcontour",
 		},
 		Spec: gatewayapi_v1alpha1.GatewaySpec{
 			Listeners: []gatewayapi_v1alpha1.Listener{{
 				Port:     80,
 				Protocol: "HTTP",
 				Routes: gatewayapi_v1alpha1.RouteBindingSelector{
+					Namespaces: gatewayapi_v1alpha1.RouteNamespaces{
+						From: "Same",
```

Comment:
I'd prefer to use the constants here and in the other gateway defs: https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha1/gateway_types.go#L448-L455

Reply:
👍